lib/src/airship_flutter.dart

```diff
@@ -586,7 +580,7 @@ class Airship {
   }
 
   /// Returns a [List] with the enabled features.
-  static Future<List<String>> getEnabledFeatures() async {
+  static FutureOr<List<String>> getEnabledFeatures() async {
```

Comment:
what is this change for?

Reply:
Allows return type to be either `Future<List<String>>` or `List<String>` while keeping type safety.

That is inline with what I saw, occasionally it would through a type related error that solved it.